docs/ReleaseNotes.md

```diff
@@ -45,7 +45,7 @@ In order to simplify typed record stores, the `FDBRecordStoreBase` class was tur
 * **Bug fix** ChainedCursor does not obey byte/record/time scan limits [(Issue #358)](https://github.com/FoundationDB/fdb-record-layer/issues/358)
 * **Bug fix** Fix 2 [(Issue #NNN)](https://github.com/FoundationDB/fdb-record-layer/issues/NNN)
 * **Bug fix** Fix 3 [(Issue #NNN)](https://github.com/FoundationDB/fdb-record-layer/issues/NNN)
-* **Bug fix** Fix 4 [(Issue #NNN)](https://github.com/FoundationDB/fdb-record-layer/issues/NNN)
+* **Bug fix** FDBRecordStore.IndexUniquenessCheck.check needs to be async [(Issue #357)](https://github.com/FoundationDB/fdb-record-layer/issues/357)
```

Comment:
`FDBRecordStore.IndexUniquenessCheck.check` should probably be in back-ticks

Reply:
Added.